src/Takeoff.cpp

```diff
@@ -99,12 +111,13 @@ unsigned char *ReadableAnsiToUChar(const char *pData, unsigned uLen);
 extern "C"
 #endif
 
-int main(int argc, char *argv[]) {
+    int
+    main(int argc, char *argv[]) {
```

Comment:
What formatter are you using?
I've seen quite a few formatting patches here and I think we should enforce a formatting standard. As you have already included quite a few formatting changes, we should use your formatter

Reply:
I am using the default formatter in VScode (ms-vscode.cpptools).